game/dont_u_panic_asshole.py

```diff
@@ -1,10 +1,12 @@
 import pygame
 import json
 import os
+
 from lib import gamestates
 from lib import intro
 from lib import login
 from lib import colors
+from lib import main_menu
 
 
 class Game(object):
```

Comment:
wywalić object, to jest zaszłość z Pythona 2, obencie nie trzeba tego robić bo z automatu dziedziczy po object